impl/src/settings.rs

```diff
@@ -626,9 +626,9 @@ fn parse_raze_settings_any_package(metadata: &Metadata) -> Result<RazeSettings>
   }
 
   // There should only be one package with raze
-  if settings_packages.len() > 1 {
+  if settings_packages.len() != 1 {
```

Comment:
It's probably worth specifying the zero case separately.  Perhaps something like:
>No raze settings were specified in the Cargo.toml file, see README.md for details on expected fields

would be more helpful?

also, not your doing, but I'm noticing that
```rust
return Err(anyhow!("error message"));
```
is used.  Anyhow provides anyhow::bail
```rust
bail!("error message");
```
that does the same thing, more concisely.

Reply:
> It's probably worth specifying the zero case separately. Perhaps something like:
> 
> > No raze settings were specified in the Cargo.toml file, see README.md for details on expected fields
> 
> would be more helpful?

Done. I do worry that it should perhaps be a URL, as if someone is just running cargo-raze as part of CI, etc, that it won't be clear which `README.md` is intended in context.

> also, not your doing, but I'm noticing that
> 
> ```rust
> return Err(anyhow!("error message"));
> ```
> 
> is used. Anyhow provides anyhow::bail
> 
> ```rust
> bail!("error message");
> ```
> 
> that does the same thing, more concisely.

Done.